setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='stability-sdk',
-    version='0.2.4',
+    version='0.2.5',
```

Comment:
Before we release a new version of the lib, we should revisit the dependency ranges that are stated below.

https://github.com/Stability-AI/stability-sdk/issues/80 raises the point that consumers may want v4 of the protobuf lib.

And generally speaking, pinning dependencies in libraries is a headache for dependency resolvers.

Therefore I propose that `install_requires` is set to the following before a new release is cut:
```python
install_requires=[
    'Pillow',
    'grpcio>=1.48.1',
    'grpcio-tools>=1.48.1',
    'python-dotenv',
    'protobuf>=3.20.2'  # same version as specified in the v0.2.4 release, but without the <4 constraint
],
```

Reply:
@wbrown David and I would be curious to hear any opinions you have on this matter

Reply:
@wbrown what sayeth ye

Reply:
I would hesitate to do that at this point. Maybe after testing, we can do this. I had issues with protocol buffers 4.